rmgpy/rmg/mainTest.py

```diff
@@ -116,6 +117,19 @@ def testRMGExecute(self):
         self.assertTrue(len(self.rmg.reactionModel.edge.reactions) > 0)
         self.assertTrue(len(self.rmg.reactionModel.edge.species) > 0)
         
+        #test seed mech generation
+        seedDir = os.path.join(originalPath,self.dir_name,'seed')
+        self.assertTrue(os.path.exists)
+        os.chdir(seedDir)
+        self.assertTrue(os.path.exists(self.rmg.name+'.py')) #thermo library made
+        self.assertTrue(os.path.exists(self.rmg.name)) #kinetics library folder made
+        kDir = os.path.join(seedDir,self.rmg.name)
+        os.chdir(kDir)
+        self.assertTrue(os.path.exists('dictionary.txt')) #dictionary file made
+        self.assertTrue(os.path.exists('reactions.py')) #reactions file made
+        
+        os.chdir(os.path.join(originalPath,self.dir_name)) #return to original directory
+        
```

Comment:
I like that we are testing that something is created. I have had a few cases where tests like this often do not check when the file written no longer functions when being read into RMG. This has in the past led to unexpected broken functionality, which is important if we want to rely more on reading from these files.

I guess there is a benefit to pushing this commit through, but writing specific tests is something to keep in mind.